src/callcc/callcc.ts

```diff
@@ -39,6 +39,11 @@ function trans(path: NodePath<t.Node>, plugins: any[]) {
 
 const visitor: Visitor = {
   Program(path: NodePath<t.Program>, state) {
+    let captureMethod = state.opts.captureMethod;
+    if (!captureMethod) {
+      captureMethod = 'eager';
+    }
+
```

Comment:
Instead of the `if`, you can say `let captureMethod = state.opts.captureMethod || 'eager'`